scripts/app.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable indent */
+
+const collection = [];
+
+function saveLocal() {
+  const acceptableString = JSON.stringify(collection);
+  localStorage.setItem('myBookList', acceptableString);
+}
+
```

Comment:
- [ ] Add a function that preserve the data (once you reload the page) with local storage.[ Check this link ](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md)about how to do it.